data/campaigns/Sceptre_of_Fire/scenarios/5_Hills_of_the_Shorbear_Clan.cfg

```diff
@@ -441,7 +446,7 @@
         [/unit]
         [message]
             speaker=narrator
-            message= _ "A brave dwarf ran out, grabbed a friendly gryphon by the sturdy neck-feathers, and hauled himself up over its back. The big animal took it all in stride, and seemed to understand what the pats and strokes of the dwarf’s hands meant."
+            message= _ "A brave dwarf ran out, grabbed it by the sturdy neck-feathers, and hauled himself up over its back. The big animal was friendly, took it all in stride, and seemed to understand what the pats and strokes of the dwarf’s hands meant."
```

Comment:
The in-game animation brings in a random gryphon. If you want to move the 'friendly' qualifier to the second sentence, I suggest reverting the 'it' to 'grabbed a gryphon by the sturdy neck-feathers...'

Reply:
I am introducing the new gryphon in the previous line of dialog that's why I use the "it" pronoun. 
It is not "_a_ gryphon" or "_some_ gryphon" but "_the_ gryphon" that I introduced in the previous message.

Whole thing plays out like this:
- Krawg says:
> "Krah!"

- Narrator message says:
> "Suddenly, the air was filled with the cries of another gryphon responding to Krawg’s call. The dwarves heard the flaps of wings and looked up to the sky to see it arriving."

- move unit fake makes a gryphon appear and move close

- Narrator message says:
> "A brave dwarf ran out, grabbed it by the sturdy neck-feathers, and hauled himself up over its back. The big animal was friendly, took it all in stride, and seemed to understand what the pats and strokes of the dwarf’s hands meant."

Reply:
Ah, I missed the new line. That makes more sense and also is good to explain how the gryphon suddenly appeared out-of-the-blue.